fairlearn/reductions/_moments/conditional_selection_rate.py

```diff
@@ -150,6 +151,44 @@ def load_data(self, X, y, **kwargs):
         super().load_data(X, y, event=_ALL, **kwargs)
 
 
+class TruePositiveRateDifference(ConditionalSelectionRate):
+    r"""Implementation of Equal Opportunity as a moment.
+
+    Adds conditioning on `y=1` compared to Demographic parity, i.e.
+
+    .. math::
+       P[h(X) = 1 | A = a, Y = 1] = P[h(X) = 1 | Y = 1] \; \forall a, y,
+
```

Comment:
This is not quite correct, we have only one event corresponding to `y = 1` now. ALso fix `index` and review the rest of the description to make sure it matches the current implementation.

Reply:
Is this the correction that was expected @MiroDudik ? Since there is only 1 value of y, it is not included in the `\forall` clause. so now it will just be `\forall a` instead of `\forall a,y`.

Reply:
looks good